demo-shell-ng2/app/components/files/files.component.html

```diff
@@ -3,10 +3,10 @@
         [rootFolderId]="documentList.currentFolderId"
         [versioning]="versioning"
         [enabled]="documentList.hasCreatePermission()">
-        <alfresco-document-list-breadcrumb
+        <adf-breadcrumb
```

Comment:
can you mark somewhere in the readme that component has two names and the new one is `adf-breadcrumb`